adafruit_funhouse/peripherals.py

```diff
@@ -90,7 +91,7 @@ def __init__(self):
         self._pir.direction = Direction.INPUT
 
     @staticmethod
-    def play_tone(frequency, duration):
+    def play_tone(frequency: float, duration: float) -> None:
```

Comment:
I think frequency might be an `int` I found a usage of this function here: https://github.com/adafruit/Adafruit_CircuitPython_FunHouse/blob/4a2868cc6d3e6ebd9d4cab61d5ae18b4f5f4a502/examples/funhouse_adafruit_io_mqtt.py#L30

Reply:
Ah got it, changed!